lib/device-sockets/ios/socket-proxy-factory.ts

```diff
@@ -119,6 +119,7 @@ export class SocketProxyFactory implements ISocketProxyFactory {
 
 		let socketFileLocation = temp.path({ suffix: ".sock" });
 		server.listen(socketFileLocation);
+		this.$logger.info("socket-file-location: " + socketFileLocation);
```

Comment:
You are right, the extension always runs tns debug with --no-client. I will fix it ;)